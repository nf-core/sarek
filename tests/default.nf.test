nextflow_pipeline {

    name "Test pipeline"
    script "../main.nf"
    tag "pipeline"
    tag "pipeline_sarek"
    tag "cpu"

    test("-profile test") {

        when {
            params {
                outdir = "$outputDir"
            }
        }

        then {
            // stable_name: All files + folders in ${params.outdir}/ with a stable name
            def stable_name = getAllFilesFromDir(params.outdir, relative: true, includeDir: true, ignore: ['pipeline_info/*.{html,json,txt}'])
            // stable_path: All files in ${params.outdir}/ with stable content
            def stable_path = getAllFilesFromDir(params.outdir, ignoreFile: 'tests/.nftignore')
            // bam_files: All bam files
            def bam_files  = getAllFilesFromDir(params.outdir, include: ['**/*.bam'])
            // cram_files: All cram files
            def cram_files  = getAllFilesFromDir(params.outdir, include: ['**/*.cram'])
            // Fasta file for cram verification with nft-bam
            def fasta_base  = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
            def fasta       = fasta_base + 'genomics/homo_sapiens/genome/genome.fasta'
            // vcf_files: All vcf files
            def vcf_files  = getAllFilesFromDir(params.outdir, include: ['**/*.vcf.gz'])
            assertAll(
                { assert workflow.success},
                { assert snapshot(
                    // Number of successful tasks
                    workflow.trace.succeeded().size(),
                    // pipeline versions.yml file for multiqc from which Nextflow version is removed because we tests pipelines on multiple Nextflow versions
                    removeNextflowVersion("$outputDir/pipeline_info/nf_core_sarek_software_mqc_versions.yml"),
                    // All stable path name, with a relative path
                    stable_name,
                    // All files with stable contents
                    stable_path,
                    // All bam files
                    bam_files.isEmpty() ? 'No BAM files' : bam_files.collect { file -> [file.getName(), bam(file.toString()).readsMD5] },
                    // All cram files
                    cram_files.collect { file -> [file.getName(), cram(file.toString(), fasta).readsMD5] },
                    // All vcf files
                    vcf_files.collect { file -> [file.getName(), path(file.toString()).vcf.variantsMD5] }
                ).match() }
            )
        }
    }

    test("-profile test --input tests/csv/3.0/sample_with_space.csv") {

        when {
            params {
                input = "${projectDir}/tests/csv/3.0/sample_with_space.csv"
                outdir = "$outputDir"
            }
        }

        then {
            assertAll(
                { assert workflow.failed},
                { assert workflow.stderr.toString().contains("Sample ID must be provided and cannot contain spaces") }
            )
        }
    }
}
