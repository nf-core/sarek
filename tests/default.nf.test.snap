{
    "-profile test": {
        "content": [
            23,
            {
                "BCFTOOLS_STATS": {
                    "bcftools": 1.21
                },
                "BWAMEM1_INDEX": {
                    "bwa": "0.7.18-r1243-dirty"
                },
                "BWAMEM1_MEM": {
                    "bwa": "0.7.18-r1243-dirty",
                    "samtools": 1.21
                },
                "CREATE_INTERVALS_BED": {
                    "gawk": "5.1.0"
                },
                "FASTQC": {
                    "fastqc": "0.12.1"
                },
                "GATK4_APPLYBQSR": {
                    "gatk4": "4.6.1.0"
                },
                "GATK4_BASERECALIBRATOR": {
                    "gatk4": "4.6.1.0"
                },
                "GATK4_INTERVALLISTTOBED": {
                    "gatk4": "4.6.1.0"
                },
                "GATK4_MARKDUPLICATES": {
                    "gatk4": "4.6.1.0",
                    "samtools": 1.21
                },
                "INDEX_CRAM": {
                    "samtools": 1.21
                },
                "MOSDEPTH": {
                    "mosdepth": "0.3.10"
                },
                "SAMTOOLS_STATS": {
                    "samtools": 1.21
                },
                "STRELKA_SINGLE": {
                    "strelka": "2.9.10"
                },
                "TABIX_BGZIPTABIX_INTERVAL_COMBINED": {
                    "tabix": 1.21
                },
                "TABIX_BGZIPTABIX_INTERVAL_SPLIT": {
                    "tabix": 1.21
                },
                "VCFTOOLS_TSTV_COUNT": {
                    "vcftools": "0.1.16"
                }
            },
            [
                "csv",
                "csv/markduplicates.csv",
                "csv/markduplicates_no_table.csv",
                "csv/recalibrated.csv",
                "csv/variantcalled.csv",
                "multiqc",
                "multiqc/multiqc_data",
                "multiqc/multiqc_data/bcftools_stats_indel-lengths.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Indels.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_SNP.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transitions.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transversions.txt",
                "multiqc/multiqc_data/fastqc-status-check-heatmap.txt",
                "multiqc/multiqc_data/fastqc_adapter_content_plot.txt",
                "multiqc/multiqc_data/fastqc_per_base_n_content_plot.txt",
                "multiqc/multiqc_data/fastqc_per_base_sequence_quality_plot.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Counts.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Percentages.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_quality_scores_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_counts_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_duplication_levels_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_length_distribution_plot.txt",
                "multiqc/multiqc_data/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.txt",
                "multiqc/multiqc_data/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.txt",
                "multiqc/multiqc_data/gatk-base-recalibrator-reported-empirical-plot.txt",
                "multiqc/multiqc_data/gatk_base_recalibrator.txt",
                "multiqc/multiqc_data/mosdepth-coverage-per-contig-single.txt",
                "multiqc/multiqc_data/mosdepth-cumcoverage-dist-id.txt",
                "multiqc/multiqc_data/mosdepth_cov_dist.txt",
                "multiqc/multiqc_data/mosdepth_cumcov_dist.txt",
                "multiqc/multiqc_data/mosdepth_perchrom.txt",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_bcftools_stats.txt",
                "multiqc/multiqc_data/multiqc_citations.txt",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_fastqc.txt",
                "multiqc/multiqc_data/multiqc_general_stats.txt",
                "multiqc/multiqc_data/multiqc_picard_dups.txt",
                "multiqc/multiqc_data/multiqc_samtools_stats.txt",
                "multiqc/multiqc_data/multiqc_software_versions.txt",
                "multiqc/multiqc_data/multiqc_sources.txt",
                "multiqc/multiqc_data/picard_MarkIlluminaAdapters_histogram.txt",
                "multiqc/multiqc_data/picard_MeanQualityByCycle_histogram.txt",
                "multiqc/multiqc_data/picard_QualityScoreDistribution_histogram.txt",
                "multiqc/multiqc_data/picard_deduplication.txt",
                "multiqc/multiqc_data/samtools-stats-dp.txt",
                "multiqc/multiqc_data/samtools_alignment_plot.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_count.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_qual.txt",
                "multiqc/multiqc_plots",
                "multiqc/multiqc_plots/pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Indels.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_SNP.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transitions.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transversions.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-status-check-heatmap.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_adapter_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_base_n_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_base_sequence_quality_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_quality_scores_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_counts_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_counts_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_duplication_levels_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_length_distribution_plot.pdf",
                "multiqc/multiqc_plots/pdf/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.pdf",
                "multiqc/multiqc_plots/pdf/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.pdf",
                "multiqc/multiqc_plots/pdf/gatk-base-recalibrator-reported-empirical-plot.pdf",
                "multiqc/multiqc_plots/pdf/general_stats_table.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-cnt.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-pct.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-cumcoverage-dist-id.pdf",
                "multiqc/multiqc_plots/pdf/picard_deduplication-cnt.pdf",
                "multiqc/multiqc_plots/pdf/picard_deduplication-pct.pdf",
                "multiqc/multiqc_plots/pdf/samtools-stats-dp.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_count.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_qual.pdf",
                "multiqc/multiqc_plots/png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Indels.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_SNP.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transitions.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transversions.png",
                "multiqc/multiqc_plots/png/fastqc-status-check-heatmap.png",
                "multiqc/multiqc_plots/png/fastqc_adapter_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_base_n_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_base_sequence_quality_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Counts.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_quality_scores_plot.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_counts_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_counts_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_duplication_levels_plot.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_length_distribution_plot.png",
                "multiqc/multiqc_plots/png/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.png",
                "multiqc/multiqc_plots/png/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.png",
                "multiqc/multiqc_plots/png/gatk-base-recalibrator-reported-empirical-plot.png",
                "multiqc/multiqc_plots/png/general_stats_table.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-cnt.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-pct.png",
                "multiqc/multiqc_plots/png/mosdepth-cumcoverage-dist-id.png",
                "multiqc/multiqc_plots/png/picard_deduplication-cnt.png",
                "multiqc/multiqc_plots/png/picard_deduplication-pct.png",
                "multiqc/multiqc_plots/png/samtools-stats-dp.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-cnt.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-pct.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_count.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_qual.png",
                "multiqc/multiqc_plots/svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Indels.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_SNP.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transitions.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transversions.svg",
                "multiqc/multiqc_plots/svg/fastqc-status-check-heatmap.svg",
                "multiqc/multiqc_plots/svg/fastqc_adapter_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_base_n_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_base_sequence_quality_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_quality_scores_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_counts_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_counts_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_duplication_levels_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_length_distribution_plot.svg",
                "multiqc/multiqc_plots/svg/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.svg",
                "multiqc/multiqc_plots/svg/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.svg",
                "multiqc/multiqc_plots/svg/gatk-base-recalibrator-reported-empirical-plot.svg",
                "multiqc/multiqc_plots/svg/general_stats_table.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-cnt.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-pct.svg",
                "multiqc/multiqc_plots/svg/mosdepth-cumcoverage-dist-id.svg",
                "multiqc/multiqc_plots/svg/picard_deduplication-cnt.svg",
                "multiqc/multiqc_plots/svg/picard_deduplication-pct.svg",
                "multiqc/multiqc_plots/svg/samtools-stats-dp.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-pct.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_count.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_qual.svg",
                "multiqc/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_sarek_software_mqc_versions.yml",
                "preprocessing",
                "preprocessing/markduplicates",
                "preprocessing/markduplicates/test",
                "preprocessing/markduplicates/test/test.md.cram",
                "preprocessing/markduplicates/test/test.md.cram.crai",
                "preprocessing/recal_table",
                "preprocessing/recal_table/test",
                "preprocessing/recal_table/test/test.recal.table",
                "preprocessing/recalibrated",
                "preprocessing/recalibrated/test",
                "preprocessing/recalibrated/test/test.recal.cram",
                "preprocessing/recalibrated/test/test.recal.cram.crai",
                "reference",
                "reports",
                "reports/bcftools",
                "reports/bcftools/strelka",
                "reports/bcftools/strelka/test",
                "reports/bcftools/strelka/test/test.strelka.variants.bcftools_stats.txt",
                "reports/fastqc",
                "reports/fastqc/test-test_L1",
                "reports/fastqc/test-test_L1/test-test_L1_1_fastqc.html",
                "reports/fastqc/test-test_L1/test-test_L1_1_fastqc.zip",
                "reports/fastqc/test-test_L1/test-test_L1_2_fastqc.html",
                "reports/fastqc/test-test_L1/test-test_L1_2_fastqc.zip",
                "reports/fastqc/test-test_L2",
                "reports/fastqc/test-test_L2/test-test_L2_1_fastqc.html",
                "reports/fastqc/test-test_L2/test-test_L2_1_fastqc.zip",
                "reports/fastqc/test-test_L2/test-test_L2_2_fastqc.html",
                "reports/fastqc/test-test_L2/test-test_L2_2_fastqc.zip",
                "reports/markduplicates",
                "reports/markduplicates/test",
                "reports/markduplicates/test/test.md.cram.metrics",
                "reports/mosdepth",
                "reports/mosdepth/test",
                "reports/mosdepth/test/test.md.mosdepth.global.dist.txt",
                "reports/mosdepth/test/test.md.mosdepth.region.dist.txt",
                "reports/mosdepth/test/test.md.mosdepth.summary.txt",
                "reports/mosdepth/test/test.md.regions.bed.gz",
                "reports/mosdepth/test/test.md.regions.bed.gz.csi",
                "reports/mosdepth/test/test.recal.mosdepth.global.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.region.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.summary.txt",
                "reports/mosdepth/test/test.recal.regions.bed.gz",
                "reports/mosdepth/test/test.recal.regions.bed.gz.csi",
                "reports/samtools",
                "reports/samtools/test",
                "reports/samtools/test/test.md.cram.stats",
                "reports/samtools/test/test.recal.cram.stats",
                "reports/vcftools",
                "reports/vcftools/strelka",
                "reports/vcftools/strelka/test",
                "reports/vcftools/strelka/test/test.strelka.variants.FILTER.summary",
                "reports/vcftools/strelka/test/test.strelka.variants.TsTv.count",
                "reports/vcftools/strelka/test/test.strelka.variants.TsTv.qual",
                "variant_calling",
                "variant_calling/strelka",
                "variant_calling/strelka/test",
                "variant_calling/strelka/test/test.strelka.genome.vcf.gz",
                "variant_calling/strelka/test/test.strelka.genome.vcf.gz.tbi",
                "variant_calling/strelka/test/test.strelka.variants.vcf.gz",
                "variant_calling/strelka/test/test.strelka.variants.vcf.gz.tbi"
            ],
            [
                "fastqc-status-check-heatmap.txt:md5,a020b9689ddeb4abec16b4854fe452f1",
                "fastqc_adapter_content_plot.txt:md5,2e1b72be741319e7fadbbb39d7e5b37d",
                "fastqc_per_base_n_content_plot.txt:md5,ad3b971a6bb4e8ba6c844c8a03584eb8",
                "fastqc_per_base_sequence_quality_plot.txt:md5,1bc03889d243a944253ac637d81ae10c",
                "fastqc_per_sequence_gc_content_plot_Counts.txt:md5,2c42d140ce06c08dad2b58f397c23239",
                "fastqc_per_sequence_gc_content_plot_Percentages.txt:md5,59e22821d350bfb97c37ffd9088f5ad9",
                "fastqc_per_sequence_quality_scores_plot.txt:md5,f33615cc98bb6225f39545a415fa7c0f",
                "fastqc_sequence_counts_plot.txt:md5,7f0f19a58e8e54e792a751fd04a9ae13",
                "fastqc_sequence_duplication_levels_plot.txt:md5,92b02e250ff78725deb9a10d510fcecc",
                "fastqc_sequence_length_distribution_plot.txt:md5,fb04dce68ec566314125bc9438211b28",
                "mosdepth-coverage-per-contig-single.txt:md5,759c540e5c35e84f2066d7a92fecf8e6",
                "mosdepth-cumcoverage-dist-id.txt:md5,f747672151fc2d3700a09133181ed208",
                "mosdepth_cov_dist.txt:md5,6769d443cfc128a729092e368ea4abed",
                "mosdepth_cumcov_dist.txt:md5,6769d443cfc128a729092e368ea4abed",
                "mosdepth_perchrom.txt:md5,759c540e5c35e84f2066d7a92fecf8e6",
                "multiqc_citations.txt:md5,ace4ca89138a5f1e2be289c157c00bd9",
                "multiqc_fastqc.txt:md5,bde0d0bffa62228b33fb68b7e25b6ff8",
                "multiqc_samtools_stats.txt:md5,dedb8d6b6d5a7ddc597975afdf5610dc",
                "picard_MarkIlluminaAdapters_histogram.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "picard_MeanQualityByCycle_histogram.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "picard_QualityScoreDistribution_histogram.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "samtools-stats-dp.txt:md5,545466eeeb8c9bfde6c119cb17392f81",
                "samtools_alignment_plot.txt:md5,717f499a3543e7ee4c7a8454bf80aeca",
                "test.strelka.variants.bcftools_stats.txt:md5,3e171488118dbc521668c1cb79414410",
                "test.md.mosdepth.global.dist.txt:md5,ef7c375ae07aec5540f9892b9b556b73",
                "test.md.mosdepth.region.dist.txt:md5,212efff2213f6fc1c3204daf68bbb8c8",
                "test.md.mosdepth.summary.txt:md5,72114393647ff64503522760218b30f0",
                "test.md.regions.bed.gz:md5,985db429051ddcd5eae177da6fb55ad6",
                "test.md.regions.bed.gz.csi:md5,3fa0f8272fefafe3cd840376d34a94a2",
                "test.recal.mosdepth.global.dist.txt:md5,ef7c375ae07aec5540f9892b9b556b73",
                "test.recal.mosdepth.region.dist.txt:md5,212efff2213f6fc1c3204daf68bbb8c8",
                "test.recal.mosdepth.summary.txt:md5,72114393647ff64503522760218b30f0",
                "test.recal.regions.bed.gz:md5,985db429051ddcd5eae177da6fb55ad6",
                "test.recal.regions.bed.gz.csi:md5,3fa0f8272fefafe3cd840376d34a94a2",
                "test.strelka.variants.FILTER.summary:md5,dd87f507da7de20d5318841af312493b",
                "test.strelka.variants.TsTv.count:md5,fa27f678965b7cba6a92efcd039f802a"
            ],
            "No BAM files",
            [
                "test.md.cram:md5,724c601c9daf019d356a53a7d5e1c8b1",
                "test.recal.cram:md5,dbd6f40b1e6d72501dc034e62e9d54eb"
            ],
            [
                "test.strelka.genome.vcf.gz:md5,1e55bf36b678e91f8ac7ee73f37815c0",
                "test.strelka.variants.vcf.gz:md5,e81855d97d9077edb8d99052ed595fd5"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.3"
        },
        "timestamp": "2025-06-12T11:30:25.583675475"
    }
}